diff -uNr WEB-INF.orig/jsp/modules/visualize.jsp WEB-INF/jsp/modules/visualize.jsp
--- WEB-INF.orig/jsp/modules/visualize.jsp	2014-11-13 09:09:28.000000000 +0100
+++ WEB-INF/jsp/modules/visualize.jsp	2015-01-07 11:16:32.000000000 +0100
@@ -16,7 +16,8 @@
 
 <c:choose>
     <c:when test="${optimizeJavascript == true}">
-        <jsp:include page="${scriptsFolderInternal}/bower_components/visualize-js/build/visualize.js" />
+        <%--jsp:include page="${scriptsFolderInternal}/bower_components/visualize-js/build/visualize.js" --%>
+		<c:import url="${scriptsFolderInternal}/bower_components/visualize-js/build/visualize.js" charEncoding="UTF-8"/>
         <%-- Workaround to fix jquery.ui.datepicker in case if optimization is enabled. In this case __jrsConfigs__ will not be global --%>
         visualize.__jrsConfigs__["userLocale"] = "${userLocale}";
         visualize.__jrsConfigs__["avaliableLocales"] = ["de", "en", "es", "fr", "it", "ja", "ro", "zh_TW", "zh_CN"];
